File: src/main/java/com/apmosys/employeeportal/dto/MigrateTeam.java
package com.apmosys.employeeportal.dto;

import java.util.List;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class MigrateTeam {
    private Integer sourceProjectId;
    private Integer targetProjectId;
    private Long targetPoId;
    private Long targetTeamId;
    private Long currentUserEmpId;
    private List<Long> migrationTeamIds;
    private List<Long> empIds;
    private List<Long> etmIds;
    private boolean mergeTeam;
    private Long targetRoleId;
}

File: src/main/frontend/src/app/models/migrateTeam.ts
export class MigrateTeams {

    sourceProjectId: any;
    targetProjectId: any;
    targetPoId: any;
    targetTeamId: any;
    targetRoleId: any;
    targetPoRequirementMappingId: any
    targetEmpTeamDepartmentId:any;
    currentUserEmpId: any;
    migrationTeamIds: any[] = [];
    mergeTeam: boolean = false;
    isAllMemberSelected: boolean = false;

    empId: any;
    projectType: any = 'Bench';
    isInternalProject: boolean = true;
    employeeRole: any;

    poDetailsList: any[] = [];
    teamList: any[] = [];
    resourceRequirementList: any[] = [];
    employeeRoles: any[] = [];
    updatedBy: any;
    empIds: any;
    etmIds: any[] = [];

}